### PR TITLE
Init V2

### PR DESCRIPTION
**Pull request by Thovi98**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by Thovi98** [comment]
!testme

**Comment by Thovi98** [comment]
!bookwormtestme

**Comment by Thovi98** [comment]
!testme

**Comment by Thovi98** [comment]
!testme

File: conf/nginx.conf
Comment (Salamandar) [inline conf/nginx.conf:8]: Did you remove robots.txt ?

File: conf/settings.py
Comment (Salamandar) [inline conf/settings.py:158]: You could name it "path_no_slash" for readability

File: conf/weblate.service
Comment (Salamandar) [inline conf/weblate.service:25]: Hmmmm not sure what’s the signification of that ?

File: manifest.toml
Comment (Salamandar) [inline manifest.toml:12]: This should be the name of a person I think. Even better, a github username :) Or an empty array.

File: manifest.toml
Comment (Salamandar) [inline manifest.toml:32]: Wow, thats’ huge :o

File: manifest.toml
Comment (Salamandar) [inline manifest.toml:56]: Not sure you know, but this can be a dict. For example:

```
choices.github = "Github"
choices.gitlab = "GitLab"
choices.another_forge = "Some string that will be translatable in the future"
```

That way, the setting / bash variable is the key (so without caps / spaces), I find it easier / less error prone.
That’s not mandatory though, it’s OK as is :)

File: manifest.toml
Comment (Salamandar) [inline manifest.toml:65]: I’m not sure a default is relevant here

File: manifest.toml
Comment (Salamandar) [inline manifest.toml:73]: Same here.

File: manifest.toml
Comment (Salamandar) [inline manifest.toml:97]: FYI that can also be written as an array 
```
packages = [ 
    "libxml2-dev",
    ...
]
```

Not mandatory. It’s OK as is.

File: conf/nginx.conf
Comment (Thovi98) [reply conf/nginx.conf:8]: Yes because I applied the default weblate.conf and there is no robot.txt in the directory

File: conf/settings.py
Comment (Thovi98) [reply conf/settings.py:158]: Oooh nice, good idea!

File: conf/weblate.service
Comment (Thovi98) [reply conf/weblate.service:25]: When service fails, it keeps saying that syslog is deprecated (or I misunderstood?)

File: manifest.toml
Comment (Thovi98) [reply manifest.toml:12]: Yes, as I said in the chat, I’m willing to take the maintainership here if needed, but I’ve no power here ^^

File: manifest.toml
Comment (Thovi98) [reply manifest.toml:32]: That’s what is said in Weblate’s installation documentation. I think it’s just an info in Yunohost or does it prevent installation on lower hardware?

File: manifest.toml
Comment (Thovi98) [reply manifest.toml:56]: Oooh nice!

File: manifest.toml
Comment (Thovi98) [reply manifest.toml:65]: It’s the CI which asked for a default ^^

File: manifest.toml
Comment (Thovi98) [inline manifest.toml:12]: @kay0u I proposed to add myself as maintainer as the package was notworking anymore seems few weeks/months and nobody seemed to care about it.
Feel free to make yourself as maintainer :)

Comment (kay0u) [reply manifest.toml:12]: I maintained this app for a while, but if you want to give some love to this package, feel free to do it!

File: conf/nginx.conf
Comment (Thovi98) [reply conf/nginx.conf:8]: @kay0u any opinion?

**Comment by Thovi98** [comment]
!testme

File: manifest.toml
Comment (kay0u) [reply manifest.toml:12]: Sure!

**Review by kay0u — COMMENTED** [review]
LGTM for the code, we should fix the upgrade before merging though

File: manifest.toml
Comment (kay0u) [reply manifest.toml:12]: Done

File: manifest.toml
Comment (Salamandar) [reply manifest.toml:33]: Nah, it's not a blocker. It's just an indication, and maybe a warning that the user needs to dismiss.

File: manifest.toml
Comment (Salamandar) [inline manifest.toml:100]:
```suggestion
    packages = [
        # Weblate dependencies
        "build-essential", "git",

        "gir1.2-pango-1.0", "libacl1-dev", "libcairo-dev", "libffi-dev", "libfreetype6-dev",
        "libgirepository1.0-dev", "libjpeg-dev", "libjpeg62-turbo-dev", "libpq-dev",
        "libssl-dev", "libxml2-dev", "libxslt-dev", "libyaml-dev", "libz-dev",
        "python3-gdbm",
        "python3-dev",
        "python3-pip",
        "python3-venv",
        "python3-virtualenv",
        "virtualenv",
        "uwsgi",
        "uwsgi-plugin-python3",
        "redis-server",
        "postgresql",
        "postgresql-contrib",
        "hub",

        # Borg backup dependencies
        "pkg-config", "python3-pkgconfig",
        "libacl1-dev", "libacl1", "libssl-dev", "liblz4-dev", "libzstd-dev", "libxxhash-dev",
    ]
```

(just some coding style)

File: scripts/install
Comment (Salamandar) [inline scripts/install:26]: Those lines are useless, you can remove them too

File: scripts/install
Comment (Salamandar) [inline scripts/install:34]: You might want to reword the message, something like "Configuring" instead of creating.

(that's not mandatory, i don't really care about messages as long as the script actually work ah ah)

**Review by Salamandar — APPROVED** [review]

**Comment by Salamandar** [comment]
About the comments i left : those are things we can fix later, once this PR is merged.

**Comment by ericgaspar** [comment]
!testme

**Review by kay0u — APPROVED** [review]
:+1:

**Pull request merged**
